config/configmap/inferenceservice.yaml

```diff
@@ -699,7 +699,7 @@ data:
 
   deploy: |-
     {
-      "defaultDeploymentMode": "Serverless"
+      "defaultDeploymentMode": "RawDeployment"
```

Comment:
why changing the default deployment mode?

Reply:
oops.. it happened.. to test .. updated.